commit 3f5278c3e5c9f4b3f85070bce6c02d37c7d2ea67
Author: Priyanshu Dangare <aspizu@protonmail.com>
Date:   Fri Oct 24 14:25:39 2025 +0530

    Add meowOS logo

diff --git a/src/logo/ascii/meowos.txt b/src/logo/ascii/meowos.txt
new file mode 100644
index 0000000..ab1a351
--- /dev/null
+++ b/src/logo/ascii/meowos.txt
@@ -0,0 +1,15 @@
+ _                        
+ \`*-.                    
+  )  _`-.                 
+ .  : `. .                
+ : _   '  \               
+ ; *` _.   `*-._          
+ `-.-'          `-.       
+   ;       `       `.     
+   :.       .        \    
+   . \  .   :   .-'   .   
+   '  `+.;  ;  '      :   
+   :  '  |    ;       ;-. 
+   ; '   : :`-:     _.`* ;
+.*' /  .*' ; .*`- +'  `*' 
+`*-*   `*-*  `*-*'        
\ No newline at end of file
diff --git a/src/logo/builtin.c b/src/logo/builtin.c
index 4de3cf0..affa585 100644
--- a/src/logo/builtin.c
+++ b/src/logo/builtin.c
@@ -3057,6 +3057,19 @@ static const FFlogo M[] = {
             FF_COLOR_FG_CYAN,
         },
     },
+    // meowOS
+    {
+        .names = {"meowOS"},
+        .lines = FASTFETCH_DATATEXT_LOGO_MEOWOS,
+        .colors = {
+            FF_COLOR_FG_MAGENTA,
+            FF_COLOR_FG_YELLOW,
+            FF_COLOR_FG_YELLOW,
+            FF_COLOR_FG_BLUE,
+        },
+        .colorKeys = FF_COLOR_FG_MAGENTA,
+        .colorTitle = FF_COLOR_FG_MAGENTA,
+    },
     // Meowix
     {
         .names = {"Meowix"},
